packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -99,9 +98,12 @@ class Product extends Component {
         })
     };
 
-    render() {
-        const { classes } = this.props;
+    addToCart = async (item, quantity) => {
+        const { guestCartId } = this.props;
+        await this.props.addItemToCart({ guestCartId, item, quantity });
```

Comment:
This `await` looks superfluous, since we're not returning a value and the caller doesn't want on the resolution